### PR TITLE
release 0.5.0

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+* Add chroot option for executing in different system root.
+* Add ENV overwrite option.
+* Allow to specify known exit codes that are not errors.
+* Document how to execute in different working directory.
+* Allow passing nil as :stdin to be same as :stdout and :strerr.
+* Convert parameters for command to strings with `.to_s`.
+* Adapt testsuite to new rspec.
+* Update documentation with various fixes.
+* Dropped support of Ruby 1.9.3.
```
Comment (lslezak) [inline CHANGELOG]: Support **for** (see ["support of" vs. "support for"](http://english.stackexchange.com/questions/91106/support-of-vs-support-for)).

**Comment by lslezak** [comment]
Otherwise LGTM

File: CHANGELOG (hunk 1 above)
Comment (ancorgs) [inline CHANGELOG]: I agree. Should be "support for". And going one step further, you use infinitive (or present) for all lines except the last two, that are in past tense. So they, for consistency, they should be:
```
* Drop support for Ruby 1.9.3.
* Add support for Ruby 2.1 and 2.2.
```
Nitpicking? Yes, but you asked :wink:

**Comment by lslezak** [comment]
LGTM

**Pull request merged**